.eslintrc.js

```diff
@@ -6,10 +6,22 @@ module.exports = {
     "plugin:markdown/recommended",
   ],
   overrides: [
+    {
```

Comment:
Can we revert the changes to the config for now? We're going to review them internally as we all have some opinions with the new rules as they are.

Reply:
The changes to the config you did in the previous PR were either fixed (so no need for the rule anymore) or were moved to be more consistent with the rest of the config + to the code they're actually applying to.